equinix/resource_metal_reserved_ip_block_acc_test.go

```diff
@@ -121,17 +124,37 @@ func TestAccMetalReservedIPBlock_public(t *testing.T) {
 
 func TestAccMetalReservedIPBlock_metro(t *testing.T) {
 	rs := acctest.RandString(10)
+	tag := "tag"
 
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: testAccMetalReservedIPBlockCheckDestroyed,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccMetalReservedIPBlockConfig_metro(rs),
+				Config: testAccMetalReservedIPBlockConfig_metro(rs, tag),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
 						"equinix_metal_reserved_ip_block.test", "metro", "sv"),
+					resource.TestCheckResourceAttr("equinix_metal_reserved_ip_block.test", "tags.0", tag),
+					resource.TestCheckResourceAttr(
+						"equinix_metal_reserved_ip_block.test", "description", "tfacc-reserved_ip_block-"+tag),
+				),
+			},
+			{
+				ResourceName:            "equinix_metal_reserved_ip_block.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"wait_for_state"},
+			},
+			{
+				Config: testAccMetalReservedIPBlockConfig_metro(rs, ""),
```

Comment:
`resourceMetalReservedIPBlockUpdate` was not being exercised (because there used to be no updateable fields and we didn't add the tests when those were introduced), so now we'll change the tags and description and expect to find the changes.